types/index.d.ts

```diff
@@ -325,6 +325,11 @@ declare namespace RamdaAdjunct {
          */
         isNotInteger(val: any): boolean;
 
+        /**
+         * Checks whether the passed value is a `bigint`.
+         */
+        isBigInt(val: any): boolean;
```

Comment:
We're missing tests for the typing. For more information how to create one please see e.g. https://github.com/char0n/ramda-adjunct/blob/bcb1f981fd46043fa1d1b3b95da00fac2d6f5b10/types/test/isSafeInteger.ts#L8